### PR TITLE
adding books from george cain's website (issue #26)

### PR DESCRIPTION
**Pull request by mdave16**
So far I've added 22 of them, I'm just keeping this open so people can see and potentially tell me where i can improve.

### CODE REVIEW

**Comment by mdave16** [comment]
Normally I would add the WIP label or something, but i may have just opened a PR too early.

**Comment by rossant** [comment]
Thanks! LGTM. Should we add a note or a link to George Cain's website? For example, a new section "Lists of textbooks" and a note along the lines of: "Some of this page's links come from the websites below:"

**Comment by mdave16** [comment]
hey @rossant, @ligurio, would you guys mind checking this PR just to see if it looks okay? All links are in the correct sections AFAIK, but I imagine I don't know everything. I also :+1: the list of links section, as many other professors have such sections.

File: README.md [hunk 1]
```diff
@@ -372,6 +440,7 @@ A curated list of awesome mathematics resources.
 * [Numerical Analysis](http://people.cs.uchicago.edu/~ridg/newna/nalrs.pdf) - L. Ridgway Scott
 * [Lectures In Basic Computational Numerical Analysis](http://www.engr.uky.edu/~acfd/egr537-lctrs.pdf) - J. M. McDonough (University of Kentucky)
 * [Advanced Numerical Methods and Their Applications to Industrial Problems: Adaptive Finite Element Methods](http://www.math.uni-bremen.de/~schmidt/SS04/YSU_Notes.pdf) - Alfred Schmidt, Arsen Narimanyan
+* [Numerical Analysis for Engineers](https://ece.uwaterloo.ca/~dwharder/NumericalAnalysis/) -Douglas Wilhelm Harder
```
Comment (rossant) [inline README.md]: missing space after `-`

File: README.md [hunk 2]
```diff
@@ -244,6 +285,7 @@ A curated list of awesome mathematics resources.
 * [General Topology](http://webusers.imj-prg.fr/~pierre.schapira/lectnotes/Topo.pdf) - Pierre Schapira (Paris VI University)
 * [Elementary Topology Problem Textbook](http://www.pdmi.ras.ru/~olegviro/topoman/eng-book-nopfs.pdf)
 * [General Topology](http://www.math.ku.dk/~moller/e03/3gt/notes/gtnotes.pdf) - Jesper M. Møller
+* [Topology Webcourse](http://at.yorku.ca/course/index.htm)
```
Comment (rossant) [inline README.md]: this link doesn't seem to work?

Comment (mdave16) [reply README.md]: good catch, i think i stopped checking the links as much closer to 60s

File: README.md [hunk 3]
```diff
@@ -363,6 +430,7 @@ A curated list of awesome mathematics resources.
 * [Applied Stochastic Processes in science and engineering](https://www.math.uwaterloo.ca/~mscott/Little_Notes.pdf) - Matt Scott (University of Waterloo)
 * [An Introduction to Stochastic Processes in Continuous Time](http://www.math.leidenuniv.nl/~spieksma/colleges/sp-master/sp-hvz1.pdf) - Flora Spieksma (Leiden University)
 * [Markov Chains and Mixing Times](http://pages.uoregon.edu/dlevin/MARKOV/markovmixing.pdf) - David A. Levin, Yuval Peres, Elizabeth L. Wilmer
+* [Convergence of Stochastic Processes](http://www.stat.yale.edu/~pollard/) - David Pollard
```
Comment (rossant) [inline README.md]: replace by http://www.stat.yale.edu/~pollard/Books/1984book/pollard1984.pdf ?

Comment (mdave16) [reply README.md]: good catch :+1:

**Comment by rossant** [comment]
Thank you for your contribution!

**Pull request merged**